test/integration/image-optimizer/test/index.test.js

```diff
@@ -0,0 +1,282 @@
+/* eslint-env jest */
+import fs from 'fs-extra'
+import { join } from 'path'
+import {
+  killApp,
+  findPort,
+  launchApp,
+  fetchViaHTTP,
+  nextBuild,
+  nextStart,
+} from 'next-test-utils'
+
+jest.setTimeout(1000 * 60 * 2)
+
+const appDir = join(__dirname, '../')
+const imagesDir = join(appDir, '.next', 'cache', 'images')
+const nextConfig = join(appDir, 'next.config.js')
+const nextConfigContent = fs.readFileSync(nextConfig, 'utf8')
```

Comment:
We can replace this with the `File` class in `next-test-utils` example here https://github.com/vercel/next.js/blob/5b1be2bb980c4cad7e543a9f150c1a1c50a6c0f3/test/integration/i18n-support/test/index.test.js#L22
```suggestion
const nextConfig = new File(join(appDir, 'next.config.js'))
```